docs/reference/metadata.md

```diff
@@ -396,31 +391,21 @@ Subfields:
 
 | Field | Cardinality |   Description   |
 |:-----:|:-----------:|:----------------|
-| ``subject`` | (1) | free text, the subject term.
-| ``identifier`` | (0-1) | the identifier of the term (``scheme`` also be specified).
-| ``scheme`` | (0-1, CV) | the subject scheme or classification code or authority.
-
-Supported subject schemes:
-
-- URL
-- URN
+| ``id`` | (1, CV) | the identifier of the subject.
 
 Example:
 
 ```json
 {
     "subjects": [{
-        "subject": "Brain",
-        "identifier": "D001921",
-        "scheme": "mesh"
-      },
-      {
-        "subject": "Invenio"
+        "id": "D001921",
       }
     ],
 }
 ```
 
+*Upcoming: Non-id bound subjects will be supported.*
```

Comment:
I marked the change as "Upcoming" rather than try to document something not coded yet.